### PR TITLE
Start adding properties to brands

### PR DESCRIPTION
**Pull request by teolemon**
Start adding properties to brands for a future constraints system

### CODE REVIEW

File: taxonomies/brands.txt [hunk 1]
```diff
@@ -2571,7 +2574,7 @@ es:Campos de Jumilla
 
 Coop. Bio Südtirol
 <es:VOG Coop. Soc. Agr.
-es:Bio Südtirol, Bio Sudtirol
+es:Südtirol, Bio Sudtirol
```
Comment (teolemon) [inline taxonomies/brands.txt:2577]:
```suggestion
es:Bio Südtirol, Bio Sudtirol
```

File: taxonomies/brands.txt [hunk 2]
```diff
@@ -2162,6 +2162,7 @@ es:Carrefour Baby
 
 <es:Carrefour S.A.
 es:Carrefour Bio
+label:en: en:Organic
```
Comment (stephanegigandet) [inline taxonomies/brands.txt:2165]: It will be easier with the exact facets ids we use (in plurals)
```suggestion
labels:en: en:organic
```

**Comment by stephanegigandet** [comment]
Looks good but at this point we don't know how the brands taxonomy will work, so it would probably be good to wait before doing too much work on it.

**Comment by teolemon** [comment]
Fixed, and also normalized other constraints that were already there

**Pull request merged**